src/main/java/bio/terra/common/logging/LoggingUtils.java

```diff
@@ -78,34 +77,30 @@ static JsonNode jsonFromString(String s) throws JsonProcessingException {
    * default Spring logging config (see resources/logback.xml) will be used.
    */
   protected static void initializeLogging(ConfigurableApplicationContext applicationContext) {
-    if (loggingInitialized) {
-      return;
-    }
-    loggingInitialized = true;
-
     ch.qos.logback.classic.Logger logbackLogger =
         (ch.qos.logback.classic.Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
     ConfigurableEnvironment environment = applicationContext.getEnvironment();
 
     if (Arrays.stream(environment.getActiveProfiles()).anyMatch("human-readable-logging"::equals)) {
       System.out.println("Human-readable logging enabled, skipping Google JSON layout");
-      return;
-    }
+      // No action needed. The logback.xml file on the classpath contains a default human-readable
```

Comment:
curious: If this branch is effectively the default behavior, why was it the one to get the keyword?